spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/storage/GcpStorageAutoConfigurationTests.java

```diff
@@ -43,10 +44,18 @@
 /**
  * @author Artem Bilan
  */
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
-		properties = "spring.cloud.gcp.storage.auto-create-files=false")
 @RunWith(SpringRunner.class)
-public class GoogleStorageAutoConfigurationTests {
+@SpringBootTest(
+		classes = {
+				GcpStorageAutoConfiguration.class,
+				GcpContextAutoConfiguration.class
+		},
+		webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+		properties = { "spring.cloud.gcp.storage.auto-create-files=false",
+				"spring.cloud.gcp.config.enabled=false",
+				"spring.cloud.gcp.sql.enabled=false"
```

Comment:
Having everything in a single `autoconfigure` module is proving challenging at testing time.
For example, because the Cloud Config stuff is being pulled into the classpath for every test, I have to disable config everywhere otherwise `GoogleConfigPropertySourceLocator` complains that "Config name must not be null".

Not specifying `GcpConfigAutoConfiguration` in `@SpringBootTest(classes)` doesn't help, as `GoogleConfigPropertySourceLocator` is still in the classpath and when the test application is booted, it tries to instantiate it.

Is there a better alternative to specifying `spring.cloud.gcp.config.enabled=false` in every test?